src/components/Drawer.js

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
+import classNames from 'classnames';
+import Drawer from 'material-ui/Drawer';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import List from 'material-ui/List';
+import { MenuItem } from 'material-ui/Menu';
+import Typography from 'material-ui/Typography';
+import TextField from 'material-ui/TextField';
+import Divider from 'material-ui/Divider';
+import IconButton from 'material-ui/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import Tooltip from 'material-ui/Tooltip';
+import CallMerge from '@material-ui/icons/CallMerge';
+
+import TrailList from './TrailList';
+import OpenLayersMap from './OpenLayersMap';
+import MapControls from './MapControls';
+import ImportExport from './ImportExport';
+
+const drawerWidth = 300;
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  paperAnchorDockedLeft: {
+    background: 'black',
+  },
+  drawerPaper: {
+    background: '#ffffffde',
+  },
+  appBar: {
+    background: '#040404',
+    position: 'absolute',
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+  },
+  appBarShift: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  },
+  'appBarShift-right': {
+    marginRight: drawerWidth,
+  },
+  menuButton: {
+    marginLeft: 12,
+    marginRight: 20,
+  },
+  hide: {
+    display: 'none',
+  },
+  drawerHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    padding: '0 8px',
+    ...theme.mixins.toolbar,
+    background: 'white'
+  }
+});
+
+
+class PersistentDrawer extends React.Component {
+  state = {
+    open: false,
+    anchor: 'left',
+  };
+
+  handleDrawerOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleDrawerClose = () => {
+    this.setState({ open: false });
+  };
+
+  handleChangeAnchor = event => {
```

Comment:
Do we need this? I think it will probably always be on the left, and since we're not using it now, one less thing to keep track of.